### PR TITLE
Updating testing-commons meeting time

### PR DESCRIPTION
**Pull request by timothysc**
/assign @spiffxp 

We should follow update policies and get a new zoom url and has open attendance and I'll need the latest hostkey.

### CODE REVIEW

**Comment by spiffxp** [comment]
/hold
I’ll get a zoom uri setup today at the new time

**Comment by timothysc** [comment]
@spiffxp ?

**Comment by spiffxp** [comment]
I made a recurring zoom meeting but need access to the meeting notes to update.  Also plan on making the invite live on a shared calendar, ref: https://github.com/kubernetes/community/issues/3345

File: sigs.yaml [hunk 1]
```diff
@@ -2003,12 +2003,11 @@ sigs:
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/OWNERS
       meetings:
       - description: Testing Commons
-        day: Wednesday
+        day: Friday
         time: "07:30"
         tz: "PT (Pacific Time)"
         frequency: bi-weekly
-        url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
-        archive_url: https://docs.google.com/document/d/1TOC8vnmlkWw6HRNHoe5xSv5-qv7LelX6XK3UVCHuwb0/edit#heading=h.tnoevy5f439o
```
Comment (spiffxp) [inline sigs.yaml]: AFAIK this needs to be flipped back to the way it was, the `archive_url` uri should link to meeting notes / agenda

**Comment by timothysc** [comment]
@spiffxp updated, plus opened the doc.

**Pull request merged**